tests/unit/models/project-test.js

```diff
@@ -1,12 +1,69 @@
 import { moduleForModel, test } from 'ember-qunit';
+import Ember from 'ember';
 
 moduleForModel('project', 'Unit | Model | project', {
   // Specify the other units that are required for this test.
-  needs: ['model:project-category', 'model:organization', 'model:task', 'model:project-skill']
+  needs: ['model:project-category', 'model:organization', 
+          'model:organization-membership', 'model:user', 
+          'model:task', 'model:project-skill']
 });
 
 test('it exists', function(assert) {
   let model = this.subject();
   // let store = this.store();
   assert.ok(!!model);
 });
+
+test('it should has attributes', function(assert) {
+  let model = this.subject();
+  let attributes = Object.keys(model.toJSON());
+  
+  assert.ok(attributes.includes('base64IconData'), 'Has base64IconData attribute');
```

Comment:
@filipecrosk is it necessary to do the second portion of the `assert.ok`? I'm not sure if it's duplicative.

Don't know what proper code style here is for writing tests. I know it may make it harder to track down because qUnit reporting isn't great. Is that the reason why?